src/main/java/core/basesyntax/dao/FruitTransactionDaoImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.db.DataBase;
+
+public class FruitTransactionDaoImpl implements FruitTransactionDao{
+    @Override
+    public void add(String fruit, int quantity) {
+        DataBase.storage.put(fruit, quantity);
+    }
+
+    @Override
+    public void update(String fruit, int quantity) {
+        DataBase.storage.put(fruit, quantity);
```

Comment:
The 'update' method is identical to the 'add' method. This could be a sign that the methods are not well thought out. Typically, 'add' would increment an existing value, while 'update' might replace it. Consider revising these methods to reflect different behaviors.